api/Document.json

```diff
@@ -11536,14 +11536,24 @@
               "version_added": "10",
               "notes": "The <code>onvisibilitychange</code> attribute is not supported in IE. To listen to this event, use <code>document.addEventListener('visibilitychange', function() {});</code>."
             },
-            "opera": {
-              "version_added": "12.1",
-              "notes": "Doesn't fire the <code>visibilitychange</code> event when the browser window is minimized, nor when <code>hidden</code> is set to true."
```

Comment:
Why remove these notes?

Reply:
I could not confirm this behavior in either Opera 12.1 or 15, nor in more recent versions of Opera (such as Opera 60).

Reply:
OK, if you can see the event firing when minimizing the browser, then dropping this note is fine. (The bit about `hidden` is misleading, it changing and the event firing is connected, both being caused by something else, like the window being hidden.)